spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1066,44 +1087,57 @@ public void setMaxHttpFormPostSize(DataSize maxHttpFormPostSize) {
 			this.maxHttpFormPostSize = maxHttpFormPostSize;
 		}
 
+		@Deprecated
+		@DeprecatedConfigurationProperty(replacement = "server.jetty.acceptors")
 		public Integer getAcceptors() {
-			return this.acceptors;
+			return this.getThreads().getAcceptors();
```

Comment:
A bit of inconsistency on naming and plurality between `Jetty#acceptors/selectors` and `Undertow#io/worker`. Should we 
a) move `acceptors -> acceptor`
b) move `io -> ios` 
c) include `num-` prefix in all of them?
d) leave alone - not a big deal